client/common/components/small/MilestoneAdd.tsx

```diff
@@ -3,22 +3,17 @@ import styles from '../../../styles/MileStoneCard.module.css';
 import { INewMilestone } from '../../types/INewMilistone';
 import { createMilestone } from '../../../pages/api/milestoneApi';
 
-const MilestoneAdd = ({project_id}: any) => {
-
+const MilestoneAdd = ({ project_id }: any) => {
   const [milestoneTitle, setMilestoneTitle] = useState('');
 
   const handleCreateMilestone = async () => {
-<<<<<<< HEAD
     if (milestoneTitle === '') return;
-=======
-
->>>>>>> development
     const newMilestone: INewMilestone = {
       title: milestoneTitle,
       project_id: 1, //! hardcoded
```

Comment:
we can grab the project_id from next's router :)

Reply:
thank you!